FixMyBerlin/fixmy.frontend
# Let administrators send notifications to accepted gastro applications

Towards https://github.com/FixMyBerlin/fixmy.platform/issues/371

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Send notice from backend admin, then follow links inside notification email to the pages for permit and traffic order
- Verify that map is visible, printable and correct
- Verify that dynamic page contents are correct, especially variations depening on the application's regulation
